Formula/gdk-pixbuf.rb

```diff
@@ -80,7 +80,7 @@ def caveats; <<-EOS.undent
     If you need to manually update the query loader cache, set these variables then run
       #{bin}/gdk-pixbuf-query-loaders --update-cache
     EOS
-  end if build.with?("relocations")
+  end if build.with?("relocations") || HOMEBREW_PREFIX.to_s != "/usr/local"
```

Comment:
This "`if build.with?("relocations")...`" looks like it's in the wrong place. This `end` appears to belong to `def caveats`. The `end` corresponding to the `if build.with?("relocations") ...` looks like it's up on line 72. Let's just remove the notations on the `end` if they're not going to be validated by `ruby`.